crt_portal/cts_forms/templates/forms/complaint_view/index/complaints_table_beta.html

```diff
@@ -0,0 +1,177 @@
+<form class="usa-form"
+      method="get"
+      action="{% url 'crt_forms:crt-forms-actions' %}"
+      novalidate
+>
+  <input type="hidden" value="{{ return_url_args }}" name="next" id="next" />
+
+  <div class="crt-xscroll">
+    <table class="usa-table crt-table sort-table">
+      <thead>
+        <tr>
+          <th>
+            <div class="usa-checkbox td-checkbox">
+              <input type="checkbox"
+                     name="all"
+                     class="usa-checkbox__input"
+                     value="all"
+                     id="checkbox-all"
+              >
+              <label class="crt-checkbox__label" for="checkbox-all" aria-label="select all records"></label>
+            </div>
+          </th>
+          <th></th>
+          {% render_sortable_heading 'Status' sort_state filter_state %}
+          {% render_sortable_heading 'Total #' sort_state filter_state nowrap=True %}
+          {% render_sortable_heading 'ID' sort_state filter_state %}
+          {% render_sortable_heading 'Routed' sort_state filter_state %}
+          {% render_sortable_heading 'Submitted' sort_state filter_state %}
+          {% render_sortable_heading 'Contact Name' sort_state filter_state %}
+          {% render_sortable_heading 'Contact Details' sort_state filter_state %}
+          {% render_sortable_heading 'Location Name' sort_state filter_state %}
+          {% render_sortable_heading 'Incident Location' sort_state filter_state %}
+          {% render_sortable_heading 'Incident Date' sort_state filter_state %}
+        </tr>
+      </thead>
+      {% if data_dict %}
+        <tbody>
+          {% for datum in data_dict %}
+            <tr class="tr-status-{{ datum.report.status }} tr--hover{% cycle ' stripe' '' %}">
+              <td>
+                <div class="usa-checkbox td-checkbox">
+                  <input type="checkbox"
+                         name="id"
+                         class="usa-checkbox__input"
+                         value="{{ datum.report.id }}"
+                         id="checkbox-{{ datum.report.id }}"
+                  >
+                  <label class="crt-checkbox__label" for="checkbox-{{ datum.report.id }}" aria-label="select record {{ datum.report.id }}"></label>
+                </div>
+              </td>
+              <td>
+                <a data-id="{{ datum.report.id }}" class="td-toggle display-flex" href="#">
+                  <img src="{% static "img/intake-icons/ic_chevron-right.svg" %}" alt="show additional information in a new row" class="icon">
+                </a>
+              </td>
+              <td>
+                <a class="td-link display-block" href="../view/{{ datum.url}}">
+                  <span class="status-tag status-{{ datum.report.status }}">
+                    {{ datum.report.status }}
+                  </span>
+                </a>
+              </td>
+              <td>
+                <a class="td-link display-block" href="{{datum.url}}">
```

Comment:
Do all these links need to have their URLs prepended with `../view/` as well? I'm noticing that each table cell has its own link element.

Reply:
> Do all these links need to have their URLs prepended with `../view/` as well? I'm noticing that each table cell has its own link element.

@louh my understanding is that that is how user can get to the detail by clicking on any of the sell of the a row instead of a specific button / link to get to the detail record. URL prepend is needed to provide/construct the source URL for "Back to All" button in the detail view.

Reply:
> Do all these links need to have their URLs prepended with `../view/` as well? I'm noticing that each table cell has its own link element.

I will see if I can break it!

Reply:
You were right @louh.  I have updated all the table elements to reflect the change.